corehq/apps/data_interfaces/app_config.py

```diff
@@ -0,0 +1,17 @@
+from django.apps import AppConfig
+
+from casexml.apps.case.models import CommCareCase
+from casexml.apps.case.signals import case_post_save
+
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.form_processor.signals import sql_case_post_save
+
+from corehq.apps.data_interfaces.signals import case_changed_receiver
+
+
+class DataInterfacesAppConfig(AppConfig):
```

Comment:
actually, looks like an easy workaround is [here](https://docs.djangoproject.com/en/1.10/topics/signals/#preventing-duplicate-signals)